include/aspect/structured_data.h

```diff
@@ -87,14 +87,41 @@ namespace aspect
          *
          * The data in @p data_table consists of a Table for each of the @p n_components components.
          *
-         * The last two arguments are rvalue references, and the function will
-         * move the data so provided into another storage location. In other
-         * words, after the call, the variables passed as the last two
-         * arguments may be empty or otherwise altered.
+         * The `coordinate_values` and `data_table` arguments are rvalue references,
+         * and the function will move the data so provided into another storage
+         * location. In other words, after the call, the variables passed as the
+         * second and third arguments may be empty or otherwise altered.
+         *
+        #if DEAL_II_VERSION_GTE(9,4,0)
```

Comment:
Why do you put this ifdef inside the comment? I don't think this will work and it makes sense to have both parts in the documentation.

Reply:
Doxygen runs (some variation of) the preprocessor before ingesting the sources. That's how our `#ifndef DOXYGEN` works.

Reply:
But macros don't work inside a comment! See https://aspect.geodynamics.org/doc/doxygen/classaspect_1_1Utilities_1_1StructuredDataLookup.html#a7a5625b138bc39bf4e53e9be4d9b3314

Reply:
Oh, interesting. On it -- see #4789.